doc/source/user/virtual_file_systems.rst

```diff
@@ -57,6 +57,25 @@ To use the .zip as a directory, you can use :file:`/vsizip/path/to/the/file.zip`
 
 Note: in the particular case where the .zip file contains a single file located at its root, just mentioning :file:`/vsizip/path/to/the/file.zip` will work.
 
+The following configuration options are specific to the /zip/ handler:
```

Comment:
@rouault the various bulleted lists of options in this file are the only significant "new" content.